astropy/wcs/include/astropy_wcs/wcslib_wrap.h

```diff
@@ -1,6 +1,5 @@
 /*
- Author: Michael Droettboom
```

Comment:
We really should remove these "authors" comments. But at least we shouldn't modify them. The code is BSD-licensed and authored by "astropy" not by individual contributors.